pkg/binlog-filter/filter.go

```diff
@@ -63,24 +63,33 @@ const (
 	DropIndex      EventType = "drop index"
 	CreateView     EventType = "create view"
 	DropView       EventType = "drop view"
-	AlertTable     EventType = "alter table"
+	AlterTable     EventType = "alter table"
+
+	CreateSchema EventType = "create schema" // alias of CreateDatabase
+	DropSchema   EventType = "drop schema"   // alias of DropDatabase
+	AddIndex     EventType = "add index"     // alias of CreateIndex
 	// if need, add more	AlertTableOption     = "alert table option"
 
 	NullEvent EventType = ""
+	// InvalidEvent is used to indicate invalid event type
+	InvalidEvent EventType = "invalid event"
```

Comment:
can we delete `NoneEvent` now?

Reply:
Sure.

Reply:
sorry, I'am wrong. We still need `NoneEvent ` to cover unsupported `ast.Stmt`.

Reply:
what's the defference of NullEvent, NoneEvent, InvalidEvent ? please add comments in code